fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirAssemblyOAuth.java

```diff
@@ -0,0 +1,112 @@
+package io.vantiq.extsrc.fhirAssembly;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Test operations against a FHIR server using fhirConnection assembly
+ */
+@Slf4j
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TestFhirAssemblyOAuth extends TestFhirAssembly {
+    
+    public static String FHIR_SOURCE = "com.vantiq.fhir.fhirServer";
+    public static String FHIR_OAUTH_SOURCE = "com.vantiq.fhir.oauthSource";
+    public static String TEST_CLIENT_ID = "accessTokenTestClient";
+    public static String TEST_CLIENT_SECRET = "9oEkhu5FKdxFiJRd2pddlT9xYwrj2aeX";
+    public static String KEYCLOAK_SOURCE_URL =
+            "http://localhost:18080/auth/realms/local/protocol/openid-connect/token/";
+    
+    // Playing this little indirection since the methods are static (Junit Rules) but we want to override.  Copying a
+    // bunch of code just to test different situations is not attractive.  JUnit's @BeforeClass will ensure that only
+    // one instance of a named method is run, so that suffices for our needs.
+    
+    @BeforeClass
+    public static void setupEnv() {
+        performSetup(getAssemblyConfigForTest());
+    }
+    
```

Comment:
Note -- this work exposed Vantiq/ag2rs#12768 & Vantiq/ag2rs#12769.  Fixes for these are required for the OAuth stuff to work.  Only the OAuth auth'n is impacted